test/src/loaders/gtdb/gtdb_genome_stats_loader_test.py

```diff
@@ -0,0 +1,72 @@
+import inspect
+import os
+import shutil
+import subprocess
+import uuid
+from pathlib import Path
+
+import jsonlines
+import pytest
+
+
+@pytest.fixture(scope="module")
+def setup_and_teardown():
+    print('starting GTDB genome statistics test')
+    tmp_dir = 'result_{}'.format(str(uuid.uuid4()))
+    os.makedirs(tmp_dir, exist_ok=True)
+
+    caller_frame = inspect.stack()[0]
+    caller_filename_full = caller_frame.filename
+    caller_file_dir = os.path.dirname(caller_filename_full)
```

Comment:
Is there a reason you can't just use `__file__` to figure out the filename?

Reply:
I think they are pretty much equivalent. Any reasons you want me to use `__file__`.

Reply:
less code, easier to understand

Reply:
If you want to delay this change as well that's fine with me

Reply:
updated to use __file__

Reply:
:+1: